operators/open_latest_version.py

```diff
@@ -19,6 +19,9 @@ def check_for_updates():
 
 def new_version_available():
     return not latest_version == VERSION
+def force_show_update():
```

Comment:
I think I would prefer a separate global variable to keep track of a broken install. Then we can show a different button message and not change the `VERSION`, which could be used for something else in the future.

Reply:
Have that changed now.

Reply:
Glanced over the part about a different button message. I'll see what I can do about that next.